administrator/components/com_installer/src/Model/UpdateModel.php

```diff
@@ -336,6 +336,23 @@ public function update($uids, $minimumStability = Updater::STABILITY_STABLE)
                 continue;
             }
 
+            $app   = Factory::getApplication();
+            $db    = $this->getDatabase();
+            $query = $db->getQuery(true)
+                ->select('type')
+                ->from('#__update_sites')
+                ->where($db->quoteName('id') . ' = :id')
+                ->bind(':id', $instance->update_site_id, ParameterType::INTEGER);
+
+            $updateSiteType = (string) $db->setQuery($query)->loadResult();
+
+            // Tuf is currently only supported for Joomla core
```

Comment:
```suggestion
            // TUF is currently only supported for Joomla core
```

Consistent naming and following the styleguide